app/web/features/profile/view/UserOverview.test.tsx

```diff
@@ -0,0 +1,119 @@
+import { render } from "@testing-library/react";
+import { HostingStatus, MeetupStatus } from "proto/api_pb";
+import wrapper from "test/hookWrapper";
+import { addDefaultUser } from "test/utils";
+
+import {
+  COMMUNITY_STANDING,
+  hostingStatusLabels,
+  meetupStatusLabels,
+  VERIFICATION_SCORE,
+} from "../constants";
+import { ProfileUserProvider } from "../hooks/useProfileUser";
+import UserOverview from "./UserOverview";
+
+describe("UserOverview", () => {
+  beforeEach(() => {
+    addDefaultUser();
+  });
+
+  describe("when user is missing", () => {
+    it("doesn't fail", () => {
+      render(<UserOverview />, { wrapper });
+    });
+  });
+
+  describe("when user is loaded and provided via context", () => {
+    it("should display the user name", () => {
+      const { getByText } = render(
+        <ProfileUserProvider user={defaultUser}>
+          <UserOverview showHostAndMeetAvailability={false} />
+        </ProfileUserProvider>,
+        { wrapper }
+      );
+      expect(getByText(defaultUser.name)).toBeInTheDocument();
```

Comment:
```suggestion
      expect(screen.getByText(defaultUser.name)).toBeInTheDocument();
```
Prefer using [screen](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen) as recommended by the RTL author rather than destructuring the result of `render` to avoid having to update the destructure if we ever change these tests. Same comment for everywhere else throughout these tests!

Reply:
Oh good to know, I wasn't aware of this recommendation! I'll update the tests. Thank you!